android/app/src/main/java/org/ocua/parity/ChooseTeams.java

```diff
@@ -16,6 +21,7 @@
 public class ChooseTeams extends Activity {
     private Context context;
     private final ChooseTeams myself = this;
+    private final int permissionRequestCode = 647662;
```

Comment:
Is the value here important or is it just a numeric identifier for the onRequestPermissionsResult callback?

Reply:
Yeah, just an identifier to associate the request with the result in the callback.